src/components/Tiles/Tile.js

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-
+import { misc } from 'js/i18n/definitions';
 import { tilePropTypes } from './proptypes';
 
-const Tile = ({ url, text, compact }) => (
-  <Link className={compact ? 'coa-Tile--compact' : 'coa-Tile'} to={url}>
-    <div className="coa-Tile__content">
-      <p className="coa-Tile__text">{text}</p>
-      <i className="material-icons">arrow_forward</i>
-    </div>
-  </Link>
-);
+import guideIcon from 'images/guide_icon.png';
+
+const Tile = ({ url, text, compact, __typename }) => {
+  console.log("misc :", misc)
+  let typename = "arrow_forward"
+  if (__typename == "GuidePageNode") {
+    typename = guideIcon
+  }
+  return (
+    <Link className={compact ? 'coa-Tile--compact' : 'coa-Tile'} to={url}>
+      <div className="coa-Tile__content">
+        <p className="coa-Tile__text">{text}</p>
+        <i className="material-icons">
```

Comment:
this logic seems a little broken, I'm thinking what we want is:
```
customTypeIcon ? <img icon etc/> : <i material arrow />
```

that way we aren't using material icon classes on images and stuff

Reply:
Went with iconType here and set the condition to be  "if" we have an icon type we'll use it "else" we'll default to the arrow.

Reply:
still don't like this, we shouldn't have an img in the material icon <i>

Reply:
```
{typeIcon ? 
    <img src={typeIcon} alt={misc.guideIcon.defaultMessage} /> :
    <i className="material-icons">arrow_forward</i>
}
```

Reply:
I see what you're getting at. Changed to that now👍